tests/test_store_policy_resolver.py

```
@@ -0,0 +1,377 @@
+            with pytest.raises(PolicyNotFound):
+                StorePolicyResolver.resolve_store_item(uuid1)
+
+            # policy2 UUID still resolves (unaffected)
+            resolved2_after = StorePolicyResolver.resolve_store_item(uuid2)
+            assert resolved2_after.price == Decimal("60.00")
+
+
+class TestFeatExactResolution:
+    """Test FEAT-STOR-004 contract: accept policy_uuid, resolve exactly."""
+
+    def test_feat_stor_004_requires_policy_uuid_not_product_id(self, app):
```

Ruff (0.16.0)

[warning] Unused method argument: `app`

(ARG002)